nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week2 homework"));
+
+app.get("/add", (req, res) => {
+
+    const first = Number(req.query.first);
+    const second = Number(req.query.second);
+    const addResult = first + second;
+    if (first && second) {
+        res.send(`add result is : ${addResult}`)
+    }
+    if (first) {
+        res.send(`you only type the firstNumber only ${first}`)
+    }
+    if (second) {
+
+        res.send(`you only type the secondNumber only${second}`)
+    }
```

Comment:
same here as below

Reply:
if (multiplyResult) {
        res.send(`add result is : ${addResult}`) 
    }else{
        res.send(`check the number you typed`)
    }

you are right. Thanks for your valuable feedback. I changed everything.